packages/uswds-core/src/styles/mixins/helpers/alert-status-styles.scss

```diff
@@ -84,10 +95,13 @@
   @include add-alert-icon($icon, $banner-text-color-token, $bgcolor);
   @include set-text-and-bg($bgcolor);
 
-  padding-left: units($theme-alert-icon-size) + ($alert-icon-optical-padding);
-  @include at-media($theme-site-margins-breakpoint) {
-    padding-left: units($theme-site-margins-width) * 2;
-    padding-right: units($theme-site-margins-width) * 2;
+  .usa-alert__heading,
+  .usa-alert__text {
+    margin-left: $alert-x-align-text-with-icon;
+
+    @include at-media($theme-site-margins-breakpoint) {
+      margin-left: $alert-x-align-text-with-icon;
+    }
```

Comment:
**Question:** Was there a reason for this media-query? It applies the same margin-left that is set in the line above it. 

<img width="495" alt="image" src="https://github.com/user-attachments/assets/48acf93f-85b1-47b8-a4e4-ead501dfa086">

Removing it doesn't appear to affect either alert.

**Suggestion:** Remove media query.

```suggestion
  .usa-alert__heading,
  .usa-alert__text {
    margin-left: $alert-x-align-text-with-icon;

    @include at-media($theme-site-margins-breakpoint) {
      margin-left: $alert-x-align-text-with-icon;
    }
```

Reply:
Good catch - removed the redundant/unnecessary media queries in [8e5295c](https://github.com/uswds/uswds/pull/5636/commits/8e5295cd51d7fdd232d96f514025210d6d987b5f)